gradle.properties

```diff
@@ -1,3 +1,4 @@
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
 org.gradle.jvmargs=-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError
+org.gradle.configuration-cache=true
```

Comment:
CC is also working for all the tasks I tested so it's also safe to enable: a little boost to productivity for this repo